File: lizhiqiang/Hamming Distance.cpp
class Solution {
public:
    int hammingDistance(int x, int y) {
        int xorNum = x ^ y;
        int count = 0;
        while (xorNum > 0) {
            xorNum = xorNum & (xorNum - 1);
            count++;
        }
        return count;
    }
};

File: mayu/HammingDistance.swift
import Foundation
class Solution {
    func hammingDistance(_ x: Int, _ y: Int) -> Int {
        let str = String(x ^ y,radix:2)
        var count =  0
        for ch in str.characters {
            count += Int("\(ch)")!
        }
        return count
    }
}
